Wabbajack.Compiler/ACompiler.cs

```diff
@@ -195,7 +197,7 @@ internal async Task<(RelativePath, AbsolutePath)> IncludeString(string str)
     public async Task<bool> GatherMetaData()
     {
         _logger.LogInformation("Getting meta data for {count} archives", SelectedArchives.Count);
-        NextStep("Gathering Metadata", SelectedArchives.Count);
+        NextStep("Building", "Gathering Metadata", SelectedArchives.Count);
```

Comment:
Should probably make the categories into constants

Reply:
Yeah, I'm not too sure where to put them however.